### DISCUSSION TITLE
Jira jql or other query languages

### DISCUSSION THREAD

**Discussion by Harryols** [grafana/grafana-infinity-datasource#539] [Specific APIs] [answered]
Hi,
How would i use Jira jql on this? It seems like "URL options">"query params" is destroying my jql input by adding & and ? when it's not needed. Where can I put my *** query language as url params undisturbed? It would be nice to have some field that could eat anything as typed in "as is".

**Comment by yesoreyeram** [comment, marked as answer]
What happen if you add params in url field itself?

**Reply by Harryols** [reply]
Well that works perfectly, my issue was that the endpoint didn't throw an auth error and then couldn't find the issue, sorry.
